sstar/init/add_star.C

```diff
@@ -88,7 +91,8 @@ void  addstar(node * b, real t_current, stellar_type type, real z,int id, bool v
 	if (b->get_starbase()->get_element_type() != NAS) return;
 	
 	//int id = b->get_index();
-        real t_cur=0, t_rel=0, m_rel=1, m_env=0, m_core=0.0;//0.01;
+        //real t_cur=0, t_rel=0, m_rel=1, m_env=0, m_core=0.0;//0.01;
+	real t_cur=0;
```

Comment:
These are now optional arguments and default values are set in `single_star.h` or `sstar_to_dyn.h`, whichever is included (I think that's always `single_star.h` but didn't check thoroughly - both use the same defaults).